chemclipse/plugins/org.eclipse.chemclipse.support.ui/src/org/eclipse/chemclipse/support/ui/workbench/PreferencesSupport.java

```diff
@@ -36,7 +41,13 @@ public static void cleanPreferencesByNodeId(List<String> preservePreferenceNodes
 	// Workaround for https://bugs.eclipse.org/bugs/show_bug.cgi?id=564022
 	public static boolean isDarkTheme() {
 
```

Comment:
I guess you mean "isDarkSystemTheme" here? If so, then I believe the new strategy should be the default one for most OSs, and Windows would be the exception as it basically doesn't set system colors.